modules/Utilities/integration.php

```diff
@@ -11,7 +11,7 @@
  * on an  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
apply only the changes you need. please undo the formatting changes in this file